common/config/chaintype.go

```diff
@@ -19,6 +19,7 @@ const (
 	ChainScroll          ChainType = "scroll"
 	ChainWeMix           ChainType = "wemix"
 	ChainXDai            ChainType = "xdai" // Deprecated: use ChainGnosis instead
+	ChainZkEvm           ChainType = "zkevm"
```

Comment:
Why is this needed?
Didn't see any behavior depending on this type.

Reply:
Oh good call, missed cleaning this out after removing the zkevm specific method